src/cli/subcommand/install.rs

```diff
@@ -160,6 +161,9 @@ impl CommandExecute for Install {
         match install_plan.install(rx1).await {
             Err(err) => {
                 if !no_confirm {
+                    // Attempt to copy self to the store if possible, but since the install failed, this might not work, that's ok.
+                    copy_self_to_nix_store().await.ok();
```

Comment:
Nit:

This still leaves around an unused `Option<T>` -- IMHO a better alternative might be:
```suggestion
                    let _ = copy_self_to_nix_store().await;
```

Reply:
I don't really have sources for you (I googled around a bit) but I believe since NLL (https://rust-lang.github.io/rfcs/2094-nll.html) is merged and stable these two statements should actually be equivalent.

Consider Pre-NLL

```rust
fn boop() {
    let snoot = vec![];

    // ... a bunch of funny business
    stuff(&snoot)
    // Snoot sticks around...
    
    // end of scope implied
    // drop(snoot);
}
```

Post NLL the compiler should infer:


```rust
fn boop() {
    let snoot = vec![];

    // ... a bunch of funny business
    stuff(&snoot)
    // lexically implied
    // drop(snoot);
    
    // end of scope implied
}
```